packages/svelte/src/internal/client/dom/elements/attributes.js

```diff
@@ -333,3 +333,22 @@ function srcset_url_equal(element, srcset) {
 		)
 	);
 }
+
+/**
+ * @param {HTMLImageElement} element
+ * @returns {void}
+ */
+export function handle_ff_lazy_img(element) {
```

Comment:
per comment below, this happens in Safari too so the function name shouldn't reference `ff`